.buildkite/pipeline.triggers.yml

```diff
@@ -0,0 +1,14 @@
+steps:
+  - label: "Triggering pipelines identity branch name"
+    trigger: "experience-identity"
+    branches: "identity/*"
```

Comment:
Is it necessary to filter on branch as well as path?

Reply:
I'd made the decision that it will _always_ build on changes in the path.

But, for whatever reason the team wanted to force a build they could use the `identity/*` branch.

Basically wanting to get it in and see what works as a process and iterate while working with Digirati.